.stylelintrc.json

```diff
@@ -15,6 +15,7 @@
     "no-empty-source": null,
     "block-no-empty": null,
     "declaration-block-no-duplicate-properties": null,
+    "selector-class-pattern": null,
```

Comment:
Is there a chance to add just exception for react-flow?

Reply:
You mean some ignore comment only for these classes?

Reply:
Yeah, this or updating the `selector-class-pattern` to allow `react-flow__`